msg_helpers.go

```diff
@@ -587,6 +587,29 @@ func unpackDataNsec(msg []byte, off int) ([]uint16, int, error) {
 	return nsec, off, nil
 }
 
+// bitMapDataLen is a helper function which computes the "maximum" length of
+// a the NSEC Type BitMap field.
+func bitMapDataLen(bitmap []uint16) int {
```

Comment:
`typeBitMapLen` is maybe slightly better, because that's what the fields in the types is called.